.github/workflows/ensure_pull_requests_are_labeled.yml

```diff
@@ -0,0 +1,28 @@
+
+name: Ensure pull requests are labeled
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled]
+
+jobs:
+  label_pull_requests:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
+            const {data: allLabels} = await github.rest.issues.listLabelsOnIssue({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+            
+            const labels = allLabels.filter(label => !label.name.includes("waiting"))
```

Comment:
Let's make sure the PR has one of the following labels:

```
"security", "feature", "bug", "perf", "refactor"
```

Reply:
To clarify this one, do you want the check to pass or fail if the PR has one of these labels, but _also_ has a "waiting-for-something" (e.g. the state of this PR right now)?

Reply:
This is a change I think from my original "spec" in the issue.

I want the check to _pass_ if the PR has one of these labels, and _fail_ if it does not have any one of these labels. Ignore waiting_ labels.